var/spack/repos/builtin/packages/simmetrix-simmodsuite/package.py

```diff
@@ -0,0 +1,336 @@
+         'msparalleladapt': ['d8882e5cb680805f1174df5ceea11414', 'paralleladapt'],
+      },
+      'docs': {
+         'MeshSimAdv': ['f5c2c0f58aeaeebbe04527d681386eeb', 'adv'],
+         'GeomSimAcis': ['9da1c3486fac8ed7deb5d65f163cd4e0', 'acis'],
+         'GeomSimImport': ['828d1db75cad7d8920471a31fb777c4b', 'import'],
+         'GeomSimSolidWorks': ['4fcdc8a9a73e1b2a65d77d6875bc5fc9', 'parasolid'],
+         'ParallelMeshSimAdapt': ['f11d765df98175c806a67a8fbc0aaf5a', 'paralleladapt'],
+         'GeomSimVoxel': ['0a20ab050ae67a9e7d8c10a3ca6d33b9', 'voxel'],
+         'GeomSim': ['3d9c30616475636f4d2c0da85684274c', 'base'],
+         'GeomSimAdvanced': ['d178a1ad442147b54d6eeb00480506dc', 'advmodel'],
+         'MeshSimAdapt': ['ea2272dd9577f8d52fd749efc33e9c15', 'base'],
+         'ParallelMeshSim': ['2729d1a03764a1fe015eac881c4ca73a', 'parallelmesh'],
+         'MeshSim': ['ea2246f55e1fcf732355afb832252b0c', 'base'],
+         'FieldSim': ['aeca95adfa92710bc6b731108124d961', 'base'],
+         'GeomSimParasolid': ['59680124b20a45fe3122f364be485398', 'parasolid'],
+         'GeomSimAbstract': ['dfe2559a7d1fb837a39afd9b24f12a26', 'abstract'],
+         'GeomSimDiscrete': ['233fa79ef98483ad69d45647ec409e48', 'discrete'],
+         'GeomSimProe': ['b2587521678eb121384d22d686b5456d', 'granite'],
+      }
+    }
+    ]
+    return releases
+
+
+def simmetrix_getKernel():
+    """only supporting the linux libraries"""
+    return "linux"
+
+
+def simmetrix_getWordSz():
+    """only supporting 64b libraries"""
+    return "64"
+
+
+def simmetrix_getLibDir(os):
+    osLibDir = {'rhel6': 'x64_rhel6_gcc44',
+                'rhel7': 'x64_rhel7_gcc48'}
+    return osLibDir[os]
+
+
+def simmetrix_makeComponentUrl(name):
+    """only supporting the linux libraries"""
+    prefix = "file://{0}/".format(os.getcwd())
+    suffix = "-" + simmetrix_getKernel() + simmetrix_getWordSz() + '.tgz'
+    return prefix + name + suffix
+
+
+def simmetrix_makeDocUrl(name):
+    """doc zip files are not os/arch specific"""
+    prefix = "file://{0}/".format(os.getcwd())
+    suffix = '.zip'
+    return prefix + name + suffix
+
+
+def simmetrix_setKernelCMakePrefixPath(spec, path, env):
+    if '+acis' in spec:
+        env.append_path('CMAKE_PREFIX_PATH', join_path(path, 'acisKrnl'))
+    if '+parasolid' in spec:
+        env.append_path('CMAKE_PREFIX_PATH', join_path(path, 'psKrnl'))
+
+
+def simmetrix_resource(name, url, md5, condition):
+    # The tarballs/zips each have the same directory structure.  Because of
+    # this, and the bug in spack described here:
+    # https://github.com/spack/spack/pull/3553#issuecomment-391424244
+    # , they cannot be expanded into the source root directory.
+    # Once this is fixed the 'destination=name' argument can be removed.
+    resource(
+      name=name,
+      url=url,
+      md5=md5,
+      destination=name,
+      when=condition
+    )
+
+
+def simmetrix_copytree(src, dst):
+    dir_util.copy_tree(src, dst)
```

Comment:
Is there any purpose to this alias? Ideally you would just use `install_tree` everywhere, but it doesn't support installing directly to the `prefix` until #8289 is merged. After it is merged, I'll probably get rid of this and just use `install_tree` everywhere.

Reply:
Replaced with calls to `install_tree`.